sympy/functions/elementary/trigonometric.py

```diff
@@ -238,6 +238,61 @@ def _pi_coeff(arg, cycles=1):
         return S.Zero
 
 
+@cacheit
+def _cospi257():
+    """ Express cos(pi/257) explicitly as a function of radicals
+        Based upon the equations in
+        http://math.stackexchange.com/questions/516142/how-does-cos2-pi-257-look-like-in-real-radicals
+        See also http://www.susqu.edu/brakke/constructions/257-gon.m.txt
```

Comment:
The link is broken

Reply:
I didn't check that during my copy-and-paste exercise... :-)

Will see if I can dig up something else.